core/src/main/scala/org/apache/spark/sql/CachedDataFrame.scala

```diff
@@ -254,6 +288,9 @@ class CachedDataFrame(df: Dataset[Row], val queryString: String,
               results(index) = resultHandler(index, r._1))
           results.iterator
       }
+      // submit cleanup job for shuffle output
+      // clearCachedShuffleDeps(sc)
+
```

Comment:
Cleanup of previous execution artifact as communicated will come subsequently. Leaving the code related to those now as is so that tried options are there. Just for convinience.